gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v1_18_0/schema-pages.yml

```diff
@@ -32,13 +32,21 @@ databaseChangeLog:
               from ${gravitee_prefix}pages
               where configuration_try_it is true
 
-        - sql:
-            comment: Move tryItURL to a the dedicated table
-            sql: >
-              insert into ${gravitee_prefix}page_configuration (page_id, k, v)
-              select id, 'tryItURL', configuration_try_it_url
-              from ${gravitee_prefix}pages
-              where configuration_try_it_url is not null
+              - sql:
```

Comment:
Updating an existing changelog is not allowed. If a change is necessary, please create a new changelog entry instead. The danger.js script should also add a comment to the PR to remind you that modifying an existing changelog is not permitted. 🤔